dev-docs/operations/iam/identity-federation-in-scs.md

```diff
@@ -13,6 +13,9 @@ to achieve that goal. To simplify connecting the different parts of SCS
 to customer owned IAM solutions, SCS deploys Keycloak as central Identity
 Provider (IdP) service.
 
+Currently this is deployed automatically only in the [osism/testbed](https://osism.github.io/docs/advanced-guides/testbed),
```

Comment:
We changed the directories to make it easier to integrate our docs in the SCS docs. Please use the new URL:

https://osism.github.io/docs/guides/deploy-guide/examples/testbed

Reply:
Maybe it makes sense to use the SCS URLs inside the SCS docs:

https://docs.scs.community/docs/iaas/guides/deploy-guide/examples/testbed

Reply:
Ok, cool, thanks for this info, how are they kept in sync? I'm also asking because I'd like to update obsolete things there, like the statement  `Keep in mind, that clicking Sign Out on the Horizon dashboard currently doesn't revoke your OIDC token` .

Reply:
You make the change in osism/osism.github.io. The SCS documentation is currently being rebuilt once a day. Then all changes from our repository are included.